thirdeye-plugins/thirdeye-oauth/src/main/java/ai/startree/thirdeye/plugins/oauth/ThirdEyePrincipal.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.plugins.oauth;
+
+import java.security.Principal;
+
+public class ThirdEyePrincipal implements Principal {
```

Comment:
This introduces a second ThirdEyePrincipal class. Is there a way we can merge these classes?

The other class https://github.com/startreedata/thirdeye/pull/1065/files#diff-2b6c0e2ccd2b52eedca9605a31255e929aeeaaf11edfee6e88c450150f92d443L18